src/components/SharedStyledComponents.js

```diff
@@ -443,3 +445,28 @@ export const dropdownIconContainerVariant = {
   },
   closed: { rotate: -90, y: 0 },
 }
+
+// Common styled item card for languages/about
+
+export const CardItem = styled(GatsbyLink)`
```

Comment:
I *believe* we can just use `Link` here vs. `GatsbyLink`. In `Link` component we check to see if language path has been explicitly set: https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/Link.js#L131